apps/angular/src/core/services/notification.service.ts

```diff
@@ -0,0 +1,24 @@
+import { inject, Injectable } from '@angular/core';
+import { MatSnackBar } from '@angular/material/snack-bar';
+
+const SNACKBAR_OPTIONS = {
+	action: 'OK',
+	duration: 7,
+};
+
+/** Notification service. */
+@Injectable({
+	providedIn: 'root',
+})
+export class NotificationService {
+	/** Snackbar instance. */
+	protected readonly _snackBar: MatSnackBar = inject(MatSnackBar);
```

Comment:
Not sure why we need it `protected` and `_`-prefix. 

Can we write it this way?
```suggestion
	/** Snackbar instance. */
	private readonly snackBar: MatSnackBar = inject(MatSnackBar);
```